libs/aws/odc/aws/queue.py

```diff
@@ -83,6 +83,39 @@ def get_queue(queue_name: str):
     return queue
 
 
+def list_queues(region: Optional[str] = None):
+    """
+    Return a list of queues which the user is allowed to see
+    """
+    sqs = boto3.resource("sqs")
+    return list(sqs.queues.all())
```

Comment:
Do you think it might be better to just use the built in generator, instead of converting it to a list?

Also, what about filtering? This has a `filter` method https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.ServiceResource.queues

Reply:
I thought using a list would be an easier way to deal with the data, once that I don't believe this function will ever return that many queues.
About the **filter**, the advantage would be to provide a **prefix** filtering the queues by that. However, the point of this function is to return all queues available to the requester.
In my case, I will get just dead queues and I'll have to filter that once I have all queues.

Reply:
Ah I see, a prefix filter is no good when we're putting a suffix on.